src/mumble/Markdown.cpp

```diff
@@ -0,0 +1,355 @@
+// Copyright 2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "Markdown.h"
+
+#include <QRegularExpression>
+#include <QRegularExpressionMatch>
+
+namespace Markdown {
+	// Placeholder constant
+	const QString regularLineBreakPlaceholder = QString::fromLatin1("%<\\!!linebreak!!//>@");
```

Comment:
`QLatin1String` is preferred over `QString::fromLatin1()`, due to better performance.

From https://doc.qt.io/qt-5/qlatin1string.html#details:
> This is a bit longer to type, but it provides exactly the same benefits as the first version of the code, and is faster than converting the Latin-1 strings using [QString::fromLatin1](https://doc.qt.io/qt-5/qstring.html#fromLatin1)().

Reply:
Every time I use that one, I find out that I need to call `arg` on it, which it doesn't support xD

And given that this string is only initialized once I think it shouldn't really matter. But I'll change it anyways as for this one I really don't need the `arg` function ^^

Reply:
Indeed, for advanced stuff you need `QString`.